test/agent/microshift-test-agent.sh

```diff
@@ -62,7 +62,7 @@ _get_current_boot_number() {
     local max_boot_attempts
 
     if test -f "${GREENBOOT_CONFIGURATION_FILE}"; then
-        # shellcheck source=/dev/null
+        # shellcheck source=test/bin/common.sh
```

Comment:
This isn't the file being sourced, right?

Reply:
Oh, that's true. For this /dev/null is what I'd expect